_databases/2026-urban-aerobiological.md

```
@@ -0,0 +1,11 @@
+---
+layout: post
+title: "Madrid Aerobiological Concentration Dataset 2026"
+type: "Experimental / Urban"
+tldr: "Timeseries and spatial distribution of pollen and pollutants in Vallecas."
+---
+
+## Database Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)